metadata-db-app/src/cmr/metadata_db/data/oracle/search.clj

```diff
@@ -94,6 +95,31 @@
          vals
          first))))
 
+(defn- append-stmt
+  "Returns the sql statement string with the given statement part properly appended to the statement."
+  [stmt-str part]
+  (let [stmt-str (string/replace stmt-str #"where null" "")]
+    (if (re-matches #"(?i).* where .*" stmt-str)
+      (format "%s and %s" stmt-str part)
+      (format "%s where %s" stmt-str part))))
+
+(defn- find-batch-starting-id-with-stmt
+  "Returns the first id that would be returned in a batch."
+  ([conn stmt]
+   (find-batch-starting-id-with-stmt conn stmt 0))
+  ([conn stmt min-id]
+   (let [stmt (-> stmt
+                  ; string/lower-case
```

Comment:
Remove the commented out code (if we keep these functions).

Reply:
Done.